.github/CODEOWNERS

```diff
@@ -55,12 +55,28 @@
 # .NET Orleans
 /docs/orleans/ @IEvangelist
 
+############### Azure ################
+
 # Azure Guide
 /docs/azure/ @alexwolfmsft @CamSoper
 
 # Azure SDK package list
 /docs/azure/includes/ @dotnet/docs
 
+# Azure SDK authentication topics
+/docs/azure/sdk/authentication/ @schaabs
+
+# Azure SDK miscellaneous topics
+/docs/azure/sdk/azure-sdk-configure-proxy.md @annelo-msft @JoshLove-msft @KrzysztofCwalina @jsquire
```

Comment:
This seems like it will get out of date really quickly. Can we add a catch-all `/docs/azure/sdk/` item above line 66 with a name like jsquire?

Reply:
Great idea. This is done.